### PR TITLE
[15.0][IMP] purchase_order_line_sequence: Show only PO related sequence in Vendor Bills

### PR DESCRIPTION
**Pull request by SergiCForgeFlow**

### CODE REVIEW

File: purchase_order_line_sequence/views/report_invoice.xml [hunk 1]
```diff
@@ -6,13 +6,15 @@
     >
         <!--complicated expr to be compatible with other modules-->
         <xpath expr="//table/thead/tr/th[1]" position="before">
-            <th t-if="any(l.related_po_sequence for l in o.invoice_line_ids)">
+            <th
+                t-if="o.move_type == 'in_invoice' and any(l.related_po_sequence for l in o.invoice_line_ids)"
```
Comment (AaronHForgeFlow) [inline purchase_order_line_sequence/views/report_invoice.xml]: What about 'in_refund' ?

Comment (SergiCForgeFlow) [reply purchase_order_line_sequence/views/report_invoice.xml]: > What about 'in_refund' ?

You are right! Just commited this.

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM! Thanks

**Review by GuillemCForgeFlow — APPROVED** [review]
LGTM 👍🏽

**Review by JordiMForgeFlow — APPROVED** [review]
Code review LGTM

**Review by MiquelRForgeFlow — APPROVED** [review]

**Pull request merged**